.gitignore

```diff
@@ -0,0 +1,4 @@
+# Git ignore list of rgw files
+build/
```

Comment:
Add one comment stating these dirs are created after build and we are good

Reply:
done